src/game.cpp

```diff
@@ -1541,6 +1541,9 @@ bool game::do_turn()
     u.process_turn();
     if( u.moves < 0 && get_option<bool>( "FORCE_REDRAW" ) ) {
         draw();
+#ifdef TILES
```

Comment:
These directives would look better inside a function. This pattern:
```c++
#ifdef TILES
        try_sdl_update();
#endif
```
occurs four times now.